### DISCUSSION TITLE
Changing scales of phi1 in ODF plots of triclinic sample symmetry

### DISCUSSION THREAD

**Discussion by trudator** [mtex-toolbox/mtex#1628] [Ask Anything] [answered]
Hi everyone,

Normally, orthorhombic is the sample symmetry that is used for conventionally produced materials (rolled, extruded, etc.), and for that reason, it is rational to minimize the ODF plots scale (e.g. from 0-90 for phi1) due to its symmetrical geometry. Does it also make sense to do the same thing for samples produced in another manufacturing process (e.g. additive manufacturing, assuming triclinic sample symmetry)? If so, how to change the range of phi1 from 0-90 in mtex? (Image below is ODF plots of triclinic sample symmetry). Thanks.

![image](https://user-images.githubusercontent.com/117386810/200195833-cacbdcd3-8508-4a2f-87e7-dc2704cbefbe.png)

**Comment by ralfHielscher** [comment, marked as answer]
As you can clearly see in your plot, restricting the region to 0..90 degree for phi1 you would miss quite some important information of your texture. On the other hand you seem to have some kine of specimen symmetry present in your data that would allow to restrict the plotting region to 0..180 degree.

Ralf.

**Reply by trudator** [reply]
Thanks Ralf. That makes sense though I'll keep it as is since the symmetry is not very clear at phi2=45.
